Changed the setbackground command

broodroosterdev opened the pull request:
To avoid spaghetti code and allow easy customizability without changing code i made it so the command would read from the shop.json file. This way adding backgrounds are going to be very easy (for the setbackground command at least)

Note: This change hasnt been fully tested yet.

teriyakisauce commented:
,
I am working on presenting the bot invite link better while not obstructing the whole README too.

broodroosterdev committed:
Changed the setbackground command

To avoid spaghetti code and allow easy customizability without changing code i made it so the command would read from the shop.json file. This way adding backgrounds are going to be very easy (for the setbackground command at least)

Note: This change hasnt been fully tested yet.
